_rules/autocomplete-valid-73f2c2.md

```diff
@@ -1,20 +1,24 @@
 ---
 id: 73f2c2
 name: Autocomplete valid
+rule_type: atomic
 description: |
   This rule checks that the HTML autocomplete attribute has a correct value
-success_criterion:
-  - 1.3.5 # Identify Input Purpose
-test_aspects:
+accessibility_requirements:
+  - wcag20: 1.3.5 # Identify Input Purpose
```

Comment:
@vemolista, actually this one has to be "WCAG21: 1.3.5 # Identify Input Purpose", but you had no way of knowing that.